code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -268,9 +268,10 @@
 			to_chat(carbon_user,"<span class='danger'>Your patrons accepts your offer..</span>")
 			var/mob/living/carbon/human/H = LH.target
 			//SKYRAT EDIT BEGIN: MAKES HERETICS NOT GIB PEOPLE
-			var/obj/item/organ/heart/heart = H.getorganslot(ORGAN_SLOT_HEART)
-			qdel(heart)
-			to_chat(H,"<span class='danger'>You feel a cold hand grasp your heart; and crush it!</span>")
+			var/obj/item/bodypart/chest/chest = H.get_bodypart(BODY_ZONE_CHEST)
```

Comment:
Can you add a check in to see if the chest is actually there? To prevent it runtiming if someone spams it or something.

Reply:
This is impossible. Right after this line of code it sets the target to null, therefor they can no longer sacrifice it.